vignettes/theme.Rmd

```diff
@@ -0,0 +1,321 @@
+      heading_font:
+        google: 
+          family: Fraunces
+          wght: "100..300"
+```
+
+
+
+### Using Bootswatch themes
+
+Using the `theme` parameter, you have access to a library of pre-made custom themes built for Bootstrap called [Bootswatch](https://bootswatch.com/). The `theme` parameter can be set to any named Bootswatch theme (in lowercase) built for [Bootstrap version 3](https://bootswatch.com/3/) or [version 4](https://bootswatch.com/).
+
+For example, here's how you could give your dashboard a [(Bootswatch 4) Minty theme](https://bootswatch.com/minty/).
+
+``` {.yaml}
+---
+output:
+  flexdashboard::flex_dashboard:
+    theme: 
+      version: 4
+      bootswatch: minty
+---
+```
+
+
+This screenshot shows the result of applying the above `theme` to [this example dashboard](https://gist.github.com/cpsievert/04bedbe9af652fab8c24a5e5eee0d721) that [uses Shiny](shiny.html) (subsequent screenshots also use this example, but swap out different `theme`s).
+
+![](images/theme-minty.png){.snippet}
+
+### Custom Bootstrap themes
+
+You can also create your own custom theme by setting the [main colors and fonts](https://rstudio.github.io/bslib/articles/theming.html#main-colors-fonts) (e.g., `bg`, `fg`, `primary`, `base_font`, `code_font`, etc.) as well as more specific theming options, [such as `navbar-bg`](https://rstudio.github.io/bslib/articles/bs4-variables.html#navbar-bg). Under the hood, this works by passing the `theme` to `bslib::bs_theme()` which influences the final look of the page by creating a custom build of Bootstrap (and Flexdashboard) CSS.
+
+``` {.r}
+---
+output:
+  flexdashboard::flex_dashboard:
+    theme:
+      version: 4
+      bg: "#101010"
+      fg: "#FDF7F7" 
+      primary: "#ED79F9"
+      navbar-bg: "#3ADAC6"
+      base_font: 
+        google: Prompt
+      code_font:
+        google: JetBrains Mono
+---
+```
+
+![](images/theme-dark.png){.snippet}
+
+As shown above, a major benefit of theming via `{bslib}` is that theming of things like `gauge()`, `valueBox()`, and even `DT::datatable()` "just work" (that is, their styling defaults are informed by the `{bslib}` theme).
+
+These components work with `{bslib}`'s real-time theming widget, which you can add to `flex_dashboard` by adding either:
+
+-   `runtime: shiny` to the document's yaml matter and calling `bslib::bs_themer()` in any code chunk, or
+
+-   `runtime: shinyrmd` to the document's yaml matter and calling `bslib::bs_themer()` in a code chunk with `context='server'`:
+
+        ```{r setup, include = FALSE, context = "server"}`r ''`
+        bslib::bs_themer()
+        ```
+
+![](images/real-time.gif){.snippet}
+
+### Theme output
+
+Generally, most "un-opinionated" HTML content rendered by the web browser "just works" with `{bslib}`. This means that the styles you specify for your document will cascade correctly. However, some HTML elements that aren't rendered by the browser aren't necessarily aware of `{bslib}` (or CSS in general).
```

Comment:
"styles you specify" sounds like custom CSS, which doesn't cascade in the same way Sass variables do

```suggestion
Generally, most "un-opinionated" HTML content rendered by the web browser should "just works" with your `theme` settings. However, some HTML elements that aren't rendered by the browser aren't necessarily aware of `{bslib}` (or CSS in general).
```